internal/internal_worker.go

```diff
@@ -73,6 +73,8 @@ const (
 	defaultMaxConcurrentSessionExecutionSize = 1000 // Large concurrent session execution size (1k)
 
 	testTagsContextKey = "cadence-testTags"
+
+	binaryChecksumEnvName = "CADENCE-CLIENT-WORKER-BINARY-CHECKSUM"
```

Comment:
Do we already expose any other configuration through the environment variable?

Reply:
Not as far as I know.How about let's remove this env variable here? 
I think the SetBinaryChecksum should be enough, the caller of worker.Start() could take care of it themselves.

Reply:
Yes, let's remove for consistency.